### PR TITLE
[Fonts] add new icon bars

### LINKED ISSUE #640: [Fonts] New icon: bars
## Description
We need to add a new icon for burger menu in the mobile header. The icon is called `bars` and is available on Fontawesome. [The Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A135&viewport=348%2C-52%2C1) has been updated with this change.

Screen Shot 2020-09-07 at 10 48 03

https://github.com/creativecommons/vocabulary/blob/8973dac8c635b018cab568d6d390c2fb9ab0195b/package.json#L1

### PR DESCRIPTION
**Pull request by rochisha0**
## Fixes

Fixes #640  by @panchovm 

## Description

I've added bars icon to the fonts.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/fonts/src/stories/vocabulary_icons.stories.mdx [hunk 1]
```diff
@@ -116,6 +116,9 @@ smallest possible file size bundle.
   <IconItem name="sliders">
     <i class="icon sliders"></i>
   </IconItem>
+  <IconItem name="bars">
+    <i class="icon bars"></i>
+  </IconItem>
```
Comment (dhruvkb) [inline packages/fonts/src/stories/vocabulary_icons.stories.mdx]: Put this below flag as it comes after flag in the order.

File: packages/fonts/src/styles/vocabulary_icons.scss [hunk 2]
```diff
@@ -59,6 +59,7 @@ $icons: (
   "filter": "\ea09",
   "sliders": "\ea0a",
   "flag": "\ea0b",
+  "bars": "\e90c",
```
Comment (dhruvkb) [inline packages/fonts/src/styles/vocabulary_icons.scss]: Code `ea0b` is followed by `ea0c`. Please reconsult [the docs](https://cc-fonts.netlify.app/?path=/docs/fonts-contribution--page).

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: packages/fonts/src/styles/vocabulary_icons.scss (hunk 2 above)
Comment (rochisha0) [reply packages/fonts/src/styles/vocabulary_icons.scss]: @dhruvkb Should I keep this with after `caret` then?

Comment (zackkrida) [reply packages/fonts/src/styles/vocabulary_icons.scss]: @rochisha0 you could move it after `caret` _or_ change the code to `ea0c` and keep it in the current position.

Comment (zackkrida) [reply packages/fonts/src/styles/vocabulary_icons.scss]: @rochisha0 wherever you move it, make sure the stories are in the same order as this file. Thank you!

**Review by zackkrida — CHANGES_REQUESTED** [review]

File: packages/fonts/src/styles/vocabulary_icons.scss (hunk 2 above)
Comment (rochisha0) [reply packages/fonts/src/styles/vocabulary_icons.scss]: @zackkrida Done with the changes!

File: packages/fonts/src/styles/vocabulary_icons.scss [hunk 3]
```diff
@@ -66,6 +66,7 @@ $icons: (
   "bookmark-regular": "\ea10",
   "bookmark-solid": "\ea11",
   "exclamation-circle": "\ea12",
+  "bars": "\e90c",
```
Comment (dhruvkb) [inline packages/fonts/src/styles/vocabulary_icons.scss:69]: @zackkrida @rochisha0 using the code `\e90c` breaks this as the `e9**` series is used for pointers.
Screenshot 2020-09-20 at 11 54 34 AM

**Review by dhruvkb — DISMISSED** [review]

File: packages/fonts/src/styles/vocabulary_icons.scss (hunk 3 above)
Comment (zackkrida) [reply packages/fonts/src/styles/vocabulary_icons.scss:69]: @rochisha0 i gave you bad advice; i hadn't read this section of the docs myself 😅 

so bars should belong to the `Symbols` section, so should go after `exclamation-circle` using the `/ea13` codepoint. Does that sound okay @dhruvkb?

Comment (dhruvkb) [reply packages/fonts/src/styles/vocabulary_icons.scss:69]: @zackkrida I just recalled our SVG icons discussion which, when implemented, renders the issue of codepoints moot altogether. I've opened a new issue #679 for that and we can probably just go with this PR as it is.

**Comment by zackkrida** [comment]
Hi @rochisha0 thanks for this! We're actually going to simplify the fonts package and switch to svgs instead of an icon font, so we're going to merge your pr, but it will be the last icon added to the icon font 😅 and we'll keep it at the `\e90c` codepoint.

Moving forward, it will be much simpler to add new icons.

**Comment by rochisha0** [comment]
@zackkrida , not sure if I was able to follow your discussion but it seems like a good change. Thanks!

**Comment by zackkrida** [comment]
😄 

@rochisha0 basically, right now fonts uses an 'icon font'. We are going to switch to providing users `.svg` files and svg code snippets that they can copy and paste.

If you were still confused about code point stuff, I'd say, forget about it! Haha

**Pull request merged**